tests/services/analysis_starter/test_configurator.py

```diff
@@ -1,36 +1,46 @@
 from pathlib import Path
 
+import mock
 import pytest
 
 from cg.services.analysis_starter.configurator.abstract_model import CaseConfig
-from cg.services.analysis_starter.configurator.abstract_service import Configurator
+from cg.services.analysis_starter.configurator.file_creators.gene_panel import GenePanelFileCreator
+from cg.services.analysis_starter.configurator.file_creators.managed_variants import (
+    ManagedVariantsFileCreator,
+)
 from cg.services.analysis_starter.configurator.implementations.nextflow import NextflowConfigurator
+from cg.services.analysis_starter.configurator.models.nextflow import NextflowCaseConfig
 
 
-@pytest.mark.parametrize(
-    "configurator_fixture, case_config_fixture, case_id_fixture",
-    [("raredisease_configurator", "raredisease_case_config", "raredisease_case_id")],
-    ids=["raredisease"],
-)
-def test_create_config(
-    configurator_fixture: str,
-    case_config_fixture: str,
-    case_id_fixture: str,
+def test_create_raredisease_config(
+    raredisease_configurator: NextflowConfigurator,
+    raredisease_case_config: NextflowCaseConfig,
+    raredisease_case_id: str,
     raredisease_params_file_path: str,
     raredisease_work_dir_path: str,
-    request: pytest.FixtureRequest,
 ):
     """Test creating the case config for all pipelines."""
     # GIVEN a configurator and a case id
-    configurator: Configurator = request.getfixturevalue(configurator_fixture)
-    case_id: str = request.getfixturevalue(case_id_fixture)
+    gene_panel_creator: GenePanelFileCreator = (
+        raredisease_configurator.pipeline_extension.gene_panel_file_creator
+    )
+    managed_variants_creator: ManagedVariantsFileCreator = (
+        raredisease_configurator.pipeline_extension.managed_variants_file_creator
+    )
+
+    # GIVEN that we mock the Scout calls
```

Comment:
```suggestion
    # GIVEN that scout returns panels and variants
```

Reply:
Not sure what the convention is but i prefer describing the actual thing we're mocking